.storybook/preview-head.html

```diff
@@ -0,0 +1 @@
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap" rel="stylesheet" />
```

Comment:
So if we are going to use the font form google and not from the local file, we could remove the local file?

Reply:
I can remove it. Is this the right solution anyway? We are using google in Terraware web - is that overridden elsewhere in Terraware web?

Reply:
I don't think it's overridden. We can merge it like this and if we think i this is the right solution, remove the local file later, because I think there are various sass file referencing the local font